com.ibm.streamsx.json/build.xml

```diff
@@ -71,12 +71,22 @@
 		</javac>
 	</target>
 
-	<target name="jar" depends="compile">
+	<target name="jar" depends="compile,standalone-jar">
 		<jar destfile="${lib.dir}/com.ibm.streamsx.json.jar" basedir="${build.dir}" 
-				 includes="com/ibm/streamsx/json/**" filesonly="true">
+				 includes="com/ibm/streamsx/json/*" filesonly="true">
 		</jar>
 		<delete dir="${build.dir}" />
 	</target>
 
+	<target name="standalone-jar" depends="compile">
```

Comment:
Minor point but since it builds the converter jar then the target name should probably reflect that.